src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts

```diff
@@ -405,16 +405,23 @@ class trafficWatch extends viewModelBase {
                         extraClass: rowHighlightRules,
                         sortable: "number"
                 }),
+                new textColumn<Raven.Client.Documents.Changes.TrafficWatchChangeBase>(grid,
+                    x => trafficWatch.isHttpItem(x) ? x.ResponseSizeInBytes : "n/a",
```

Comment:
why can't we format to bytes directly ?

Reply:
w/o using transform value

Reply:
I need to sort by bytes but display the formatted value (which cannot be sorted by string).
`30 Bytes` > `1 Kbytes` when sorting by string